plug-in-development/index.html

```diff
@@ -53,7 +53,7 @@ <h3 class="section" indent="1" id="launch-configuration">Launch configuration</h
 </p>
 <pre class="brush:java">
 fun main(argv: Array&lt;String&gt;) {
-    com.beust.kobalt.main(argv)
+    com.beust.kobalt.internal.main(argv)
```

Comment:
Is this current API or are you planning on moving ```main(argv)``` there? Your code shows up in red in IDEA as my plugin currently only depends on the official plugin API.